app/resources/person_resource.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2022, Schweizer Wanderwege. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+class PersonResource < ApplicationResource
+  primary_endpoint 'people', [:index, :show, :update]
+
+  attribute :first_name, :string
+  attribute :last_name, :string
+  attribute :nickname, :string
+  attribute :company_name, :string
+  attribute :company, :boolean
+  attribute :email, :string
+  attribute :address, :string
+  attribute :zip_code, :string
+  attribute :town, :string
+  attribute :country, :string
+  attribute :gender, :string, readable: :show_details?, writable: :write_details?
+  attribute :birthday, :date, readable: :show_details?, writable: :write_details?
+  attribute :primary_group_id, :integer, except: [:writeable]
+
+  has_many :phone_numbers,
+    link: false,
+    resource: PhoneNumberResource,
+    readable: :show_details_or_public?
+  has_many :social_accounts,
+    link: false,
+    resource: SocialAccountResource,
+    readable: :show_details_or_public?
+  has_many :additional_emails,
+    link: false,
+    resource: AdditionalEmailResource,
+    readable: :show_details_or_public?
+
+  has_many :roles,
+    link: false,
+    resource: RoleResource,
+    readable: :show_full?,
+    writable: false
```

Comment:
Bitte Rollen vorerst noch nicht spontan schreibbar machen. Ich möchte dann noch testen, ob [Traversal Attacks](https://blog.morethancode.dev/preventing-traversal-attacks-in-your-graphql-api/) möglich sind, bevor wir das aktivieren.

Reply:
Rollen sollen vorerst read-only sein !